Xamarin.Forms.Core/RadioButton.cs

```diff
@@ -0,0 +1,425 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Windows.Input;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+	[RenderWith(typeof(_RadioButtonRenderer))]
+	public partial class RadioButton : IToggleButtonElement, IToggleButtonController, ITextButtonElement
+	{
+		static Dictionary<string, List<WeakReference<RadioButton>>> _groupNameToElements;
+
+		public static readonly BindableProperty IsCheckedProperty = ToggleButtonElement.IsCheckedProperty;
+
+		public static readonly BindableProperty GroupNameProperty = BindableProperty.Create(nameof(GroupName), typeof(string), typeof(RadioButton), null,
```

Comment:
what's the scope of GroupName ? it looks global, which will prevent having a custom control embedding a group displayed twice in the same app

Reply:
@StephaneDelcroix  I tried to describe the scope of GroupName in the description. Does it make sense?